persistence/src/commonMain/db_user/com/wire/kalium/persistence/Messages.sq

```diff
@@ -221,6 +222,17 @@ CREATE TABLE MessageConversationProtocolChangedContent (
       PRIMARY KEY (message_id, conversation_id)
 );
 
+CREATE TABLE MessageLegalHoldContent (
```

Comment:
The name is a bit confusing, maybe it's better to rename it indicate that its for system messages ? `SystemMessageLegalHoldContent` ?

Reply:
It's the same as all other tables for system messages content, like for instance:
`MessageMemberChangeContent`,
`MessageFederationTerminatedContent`, 
`MessageFailedToDecryptContent`, 
`MessageMissedCallContent`, 
`MessageConversationChangedContent`
so I wouldn't change the name just for this one to keep it consistent